src/navigation/_DevSiteNavigation.jsx

```diff
@@ -95,6 +92,33 @@ class DevSiteNavigation extends React.Component {
     });
   }
 
+  static launchAppSwitcher(key, { disclosureManager, siteConfig }) {
```

Comment:
Should this logic (and the apps config object) be pushed down to the SettingsPicker layer?